loops_labs.js

```diff
@@ -0,0 +1,231 @@
+# Loops Lab
+
+To complete this lab, do the following:
+
+1. Fork this repo
+2. Clone **your** forked repo
+3. Create a file called `loops_lab.js` inside your cloned repo
+4. Copy the description of the exercise as a comment
+5. Underneath the commented description write your code.
+6. Ensure your solutions work properly!
+7. Open PR against this repo.
+
+___
+
+## Exercises
+
+1. Write a `while` loop and a `for` loop that takes the variable
+"num" and logs all the numbers, descending, between "num" and 1.
+
+// ANSWER:
+//
+//  FOR:
+
+// for (let num = 50; num > 0; num -= 1) {
+//   console.log(num);
+// }
+
+// LET :
+
+// let num = 50;
+// while (num > 0) {
+//   console.log(num);
+//   num -= 1;
+// }
+
+2. Write a `while` loop and a `for` loop that takes the variable
+"num", and iterates over all numbers from 0 to "num". For each iteration,
+it will check if the current number is even or odd, and log that
+to the screen (e.g. "2 is even")
+
+// ANSWER:
+// for (let max = 0; max <= 10; max += 1) {
+//   if (max % 2 === 0) {
+//     console.log(max + " is even")
+//   } else {
+//     console.log(max + " is odd")
+//   }
+// }
+
+//SEEKING HEEEELPPPPP!!!!
+
+// let max = 0;
+// while (max <= 10) {
+//   if (max % 2 === 0) {
+//   console.log(max + " is even")
+// } else {
+//   console.log(max + " is odd")
+// }
+//   max += 1;
+// }
+
+
+3. Write a `while` loop and a `for` loop that takes the
+variable "num" and iterates over all numbers from 0 to "num".
+For each iteration of the loop, it will multiply the
+number by 9 and log the result (e.g. "2 * 9 = 18").
+   _Bonus_ think of another way to solve it.
+     <details>
+       <summary>
+         Hint
+       </summary>
+       Find the final number and increment the loop by 9.
+     </details>
+
+ FOR LOOP:
+ for (let num = 0; num <= 10; num += 1) {
+   console.log(num + " * 9 is " + num * 9);
+ }
+
+// WHILE LOOP:
+
+// let num = 0
+// while (num < 10) {
+//   console.log(num + " * 9 " + num * 9)
+//   num ++
+// }
+
+4.  Write code that prints/logs all the numbers
+that end in a 5 from 1 to 100, exclusive.
+
+// for (let num = 5; num <= 100; num += 5) {
```

Comment:
This solution works if we are starting from 5 (let num = 5) but wouldn't work if we start from 1 (let num = 1). How can we fix this? Perhaps we can use modulo 5 and modulo 10? Think about it when you see similar question next time.